react/src/index.js

```diff
@@ -192,86 +165,85 @@ class App extends Component {
     console.log(`> backendType: ${backendType} | backendUrl: ${BACKEND_URL}`);
 
     // These also get passed via request headers (see window.fetch below)
-    Sentry.configureScope((scope) => {
-      const customerType = [
-        'medium-plan',
-        'large-plan',
-        'small-plan',
-        'enterprise',
-      ][Math.floor(Math.random() * 4)];
-      scope.setTag('customerType', customerType);
-
-      if (queryParams.get('se')) {
-        // Route components (navigation changes) will now have 'se' tag on scope
-        console.log('> src/index.js se', queryParams.get('se'));
-        scope.setTag('se', queryParams.get('se'));
-        // for use in Checkout.js when deciding whether to pre-fill form
-        // lasts for as long as the tab is open
-        sessionStorage.setItem('se', queryParams.get('se'));
-      }
-
-      if (queryParams.get('frontendSlowdown') === 'true') {
-        console.log('> frontend-only slowdown: true');
-        FRONTEND_SLOWDOWN = true;
-        scope.setTag('frontendSlowdown', true);
-      } else {
-        console.log('> frontend + backend slowdown');
-        scope.setTag('frontendSlowdown', false);
-      }
-
-      if (queryParams.get('rageclick') === 'true') {
-        RAGECLICK = true;
-      }
+    const scope = Sentry.getCurrentScope();
+    const customerType = [
+      'medium-plan',
+      'large-plan',
+      'small-plan',
+      'enterprise',
+    ][Math.floor(Math.random() * 4)];
+    scope.setTag('customerType', customerType);
```

Comment:
let's not use current scope here, and just do `Sentry.setTag(...)` etc!

Reply:
Updated and here and the other places that used this scope.